diff -Nur sqlite3-3.5.4.orig/sqlite3.pc.in sqlite3-3.5.4/sqlite3.pc.in
--- sqlite3-3.5.4.orig/sqlite3.pc.in	2007-11-30 03:28:10.000000000 +0200
+++ sqlite3-3.5.4/sqlite3.pc.in	2007-12-22 21:50:53.137612509 +0200
@@ -7,7 +7,7 @@
 
 Name: SQLite
 Description: SQL database engine
-Version: @VERSION@
+Version: @RELEASE@
 Libs: -L${libdir} -lsqlite3
 Libs.private: -lpthread
 Cflags: -I${includedir}
diff -Nur sqlite3-3.5.4.orig/sqlite.pc.in sqlite3-3.5.4/sqlite.pc.in
--- sqlite3-3.5.4.orig/sqlite.pc.in	2005-04-24 00:45:24.000000000 +0200
+++ sqlite3-3.5.4/sqlite.pc.in	2007-12-22 21:51:10.386595472 +0200
@@ -7,6 +7,6 @@
 
 Name: SQLite
 Description: SQL database engine
-Version: @VERSION@
+Version: @RELEASE@
 Libs: -L${libdir} -lsqlite
 Cflags: -I${includedir}
