diff -Nur sqlite3-3.5.8.orig/Makefile.in sqlite3-3.5.8/Makefile.in
--- sqlite3-3.5.8.orig/Makefile.in	2008-04-30 22:36:01.000000000 +0200
+++ sqlite3-3.5.8/Makefile.in	2008-04-30 22:41:33.790003485 +0200
@@ -684,6 +684,191 @@
 		-DSQLITE_SERVER=1 -DSQLITE_PRIVATE="" -DSQLITE_CORE \
 		$(TEMP_STORE) -o $@ $(TESTFIXTURE_SRC) $(LIBTCL)
 
+# Rules used to build documentation
+#
+34to35.html:     $(TOP)/www/34to35.tcl
+	tclsh $(TOP)/www/34to35.tcl >34to35.html
+
+arch.html:     $(TOP)/www/arch.tcl
+	tclsh $(TOP)/www/arch.tcl >arch.html
+
+arch2.gif:     $(TOP)/www/arch2.gif
+	cp $(TOP)/www/arch2.gif .
+
+autoinc.html:  $(TOP)/www/autoinc.tcl
+	tclsh $(TOP)/www/autoinc.tcl >autoinc.html
+
+c_interface.html:      $(TOP)/www/c_interface.tcl
+	tclsh $(TOP)/www/c_interface.tcl >c_interface.html
+
+capi3.html:    $(TOP)/www/capi3.tcl
+	tclsh $(TOP)/www/capi3.tcl >capi3.html
+
+capi3ref.html: $(TOP)/www/mkapidoc.tcl sqlite3.h
+	tclsh $(TOP)/www/mkapidoc.tcl <sqlite3.h >capi3ref.html
+
+changes.html:  $(TOP)/www/changes.tcl
+	tclsh $(TOP)/www/changes.tcl >changes.html
+
+compile.html:  $(TOP)/www/compile.tcl
+	tclsh $(TOP)/www/compile.tcl >compile.html
+
+copyright.html:        $(TOP)/www/copyright.tcl
+	tclsh $(TOP)/www/copyright.tcl >copyright.html
+
+copyright-release.html:        $(TOP)/www/copyright-release.html
+	cp $(TOP)/www/copyright-release.html .
+
+copyright-release.pdf: $(TOP)/www/copyright-release.pdf
+	cp $(TOP)/www/copyright-release.pdf .
+
+common.tcl:    $(TOP)/www/common.tcl
+	cp $(TOP)/www/common.tcl .
+
+conflict.html: $(TOP)/www/conflict.tcl
+	tclsh $(TOP)/www/conflict.tcl >conflict.html
+
+datatypes.html:        $(TOP)/www/datatypes.tcl
+	tclsh $(TOP)/www/datatypes.tcl >datatypes.html
+
+datatype3.html:        $(TOP)/www/datatype3.tcl
+	tclsh $(TOP)/www/datatype3.tcl >datatype3.html
+
+different.html:        $(TOP)/www/different.tcl
+	tclsh $(TOP)/www/different.tcl >different.html
+
+docs.html:     $(TOP)/www/docs.tcl
+	tclsh $(TOP)/www/docs.tcl >docs.html
+
+download.html: $(TOP)/www/download.tcl
+	mkdir -p doc
+	tclsh $(TOP)/www/download.tcl >download.html
+
+faq.html:      $(TOP)/www/faq.tcl
+	tclsh $(TOP)/www/faq.tcl >faq.html
+
+fileformat.html:       $(TOP)/www/fileformat.tcl
+	tclsh $(TOP)/www/fileformat.tcl >fileformat.html
+
+formatchng.html:       $(TOP)/www/formatchng.tcl
+	tclsh $(TOP)/www/formatchng.tcl >formatchng.html
+
+index.html:    $(TOP)/www/index.tcl last_change
+	tclsh $(TOP)/www/index.tcl >index.html
+
+limits.html:   $(TOP)/www/limits.tcl last_change
+	tclsh $(TOP)/www/limits.tcl >limits.html
+
+lang.html:     $(TOP)/www/lang.tcl
+	tclsh $(TOP)/www/lang.tcl >lang.html
+
+pragma.html:   $(TOP)/www/pragma.tcl
+	tclsh $(TOP)/www/pragma.tcl >pragma.html
+
+lockingv3.html:        $(TOP)/www/lockingv3.tcl
+	tclsh $(TOP)/www/lockingv3.tcl >lockingv3.html
+
+oldnews.html:  $(TOP)/www/oldnews.tcl
+	tclsh $(TOP)/www/oldnews.tcl >oldnews.html
+
+omitted.html:  $(TOP)/www/omitted.tcl
+	tclsh $(TOP)/www/omitted.tcl >omitted.html
+
+opcode.html:   $(TOP)/www/opcode.tcl $(TOP)/src/vdbe.c
+	tclsh $(TOP)/www/opcode.tcl $(TOP)/src/vdbe.c >opcode.html
+
+optoverview.html:     $(TOP)/www/optoverview.tcl
+	tclsh $(TOP)/www/optoverview.tcl >optoverview.html
+
+mingw.html:    $(TOP)/www/mingw.tcl
+	tclsh $(TOP)/www/mingw.tcl >mingw.html
+
+nulls.html:    $(TOP)/www/nulls.tcl
+	tclsh $(TOP)/www/nulls.tcl >nulls.html
+
+quickstart.html:     $(TOP)/www/quickstart.tcl
+	tclsh $(TOP)/www/quickstart.tcl >quickstart.html
+
+shared.gif:    $(TOP)/www/shared.gif
+	cp $(TOP)/www/shared.gif shared.gif
+
+sharedcache.html:    $(TOP)/www/sharedcache.tcl
+	tclsh $(TOP)/www/sharedcache.tcl >sharedcache.html
+
+speed.html:    $(TOP)/www/speed.tcl
+	tclsh $(TOP)/www/speed.tcl >speed.html
+
+sqlite.gif:    $(TOP)/art/SQLite.gif
+	cp $(TOP)/art/SQLite.gif sqlite.gif
+
+sqlite.html:   $(TOP)/www/sqlite.tcl
+	tclsh $(TOP)/www/sqlite.tcl >sqlite.html
+
+support.html:  $(TOP)/www/support.tcl
+	tclsh $(TOP)/www/support.tcl >support.html
+
+tclsqlite.html:        $(TOP)/www/tclsqlite.tcl
+	tclsh $(TOP)/www/tclsqlite.tcl >tclsqlite.html
+
+vdbe.html:     $(TOP)/www/vdbe.tcl
+	tclsh $(TOP)/www/vdbe.tcl >vdbe.html
+
+version3.html: $(TOP)/www/version3.tcl
+	tclsh $(TOP)/www/version3.tcl >version3.html
+
+whentouse.html: $(TOP)/www/whentouse.tcl
+	tclsh $(TOP)/www/whentouse.tcl >whentouse.html
+
+# Files to be published on the website.
+#
+DOC = \
+  34to35.html \
+  arch.html \
+  arch2.gif \
+  autoinc.html \
+  c_interface.html \
+  capi3.html \
+  capi3ref.html \
+  changes.html \
+  compile.html \
+  copyright.html \
+  copyright-release.html \
+  copyright-release.pdf \
+  conflict.html \
+  datatypes.html \
+  datatype3.html \
+  different.html \
+  docs.html \
+  download.html \
+  faq.html \
+  fileformat.html \
+  formatchng.html \
+  index.html \
+  lang.html \
+  limits.html \
+  lockingv3.html \
+  mingw.html \
+  nulls.html \
+  oldnews.html \
+  omitted.html \
+  opcode.html \
+  optoverview.html \
+  pragma.html \
+  quickstart.html \
+  shared.gif \
+  sharedcache.html \
+  speed.html \
+  sqlite.gif \
+  sqlite.html \
+  support.html \
+  tclsqlite.html \
+  vdbe.html \
+  version3.html \
+  whentouse.html
+
+doc:   common.tcl $(DOC)
+	mkdir -p doc
+	mv $(DOC) doc
 
 install:	sqlite3$(BEXE) libsqlite3.la sqlite3.h ${HAVE_TCL:1=tcl_install}
 	$(INSTALL) -d $(DESTDIR)$(libdir)
